src/main/kotlin/com/terraformation/backend/tracking/db/PlantingSiteStore.kt

```diff
@@ -1669,6 +1671,93 @@ class PlantingSiteStore(
     }
   }
 
+  fun migrateSimplePlantingSites(): List<String> {
+    // These are the names we use in terraware-web when creating a new planting site without
+    // specifying zones or subzones. They are not translated into the user's language.
+    val zoneName = "Zone 01"
+    val subzoneName = "Subzone A"
+
+    val messages = mutableListOf<String>()
+
+    dslContext.transaction { _ ->
+      val simplePlantingSiteIds =
+          dslContext
+              .select(PLANTING_SITES.ID)
+              .distinctOn(PLANTING_SITES.ID)
+              .from(PLANTING_SITES)
+              .where(PLANTING_SITES.BOUNDARY.isNotNull)
+              .andNotExists(
+                  DSL.selectOne()
+                      .from(PLANTING_ZONES)
+                      .where(PLANTING_ZONES.PLANTING_SITE_ID.eq(PLANTING_SITES.ID)))
+              .orderBy(PLANTING_SITES.ID)
+              .fetch(PLANTING_SITES.ID.asNonNullable())
+
+      for (siteId in simplePlantingSiteIds) {
+        try {
+          val now = clock.instant()
+
+          log.info("Migrating simple planting site $siteId to detailed")
+
+          val existingSite = fetchSiteById(siteId, PlantingSiteDepth.Site)
+          val boundary = existingSite.boundary!!.toMultiPolygon()
+
+          // This calculates the grid origin and area.
+          val newSite =
+              NewPlantingSiteModel.create(
+                  boundary,
+                  existingSite.description,
+                  name = existingSite.name,
+                  organizationId = existingSite.organizationId,
+              )
+
+          if (newSite.areaHa == null || newSite.areaHa.equalsIgnoreScale(BigDecimal.ZERO)) {
+            messages.add("Planting site $siteId (${existingSite.name}) area too small to convert")
+            continue
+          }
```

Comment:
Confirming that we're fine to convert simple planting sites that might only be 1 hectare? Just wanting to make sure that we don't need a lower limit instead of just non-zero.

Reply:
The nonzero thing is just because there are some statistics that we calculate by dividing by area, and we round area to 1 decimal place. Anything greater than 0.05 hectares is fine since it won't cause divide-by-zero errors.

Reply:
Got it. Then just to be nitpicky, doesn't `equalsIgnoreScale` mean that it's going to reject simple plots with an area of < 1, instead of < 0.1?